File: src/main/java/plus/maa/backend/controller/response/CommentsAreaInfo.java
package plus.maa.backend.controller.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;


/**
 * @author LoMu
 * Date  2023-02-19 11:47
 */

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CommentsAreaInfo {
    private String id;
    private String copilotId;
    private int like;
    private Date uploadTime;
}

File: src/main/kotlin/plus/maa/backend/controller/response/comments/CommentsAreaInfo.kt
package plus.maa.backend.controller.response.comments

/**
 * @author LoMu
 * Date  2023-02-19 11:47
 */
data class CommentsAreaInfo(
    val hasNext: Boolean,
    /**
     * Total number of pages
     */
    val page: Int,
    /**
     * Total number of elements
     */
    val total: Long,
    val data: List<CommentsInfo>,
)
